libs/product/driver/magento/src/product.service.spec.ts

```diff
@@ -98,4 +99,46 @@ describe('Product | Magento | ProductService', () => {
       });
     });
   });
+
+  describe('getByUrl | getting a single product by url', () => {
```

Comment:
these tests don't look quite right. The first one tests the `get` method, not the `getByUrl`. The second says it is handling different product types, but does a simple product again, same as the first test

Reply:
fixed the first case and removed the different product types case since that's not really in the logic of the service.